base/library/protected-annotation/src.rego

```diff
@@ -0,0 +1,17 @@
+package kubernetes.validating.annotation
+
+violation[msg] {
```

Comment:
```suggestion
violation[{"msg": msg}] {
```
I can't find this documented but I _think_ the msg field is a requirement for Gatekeeper. Certainly all the examples I can find use it.